token/jwt.go

```diff
@@ -293,7 +303,7 @@ func (j *Service) Get(r *http.Request) (Claims, string, error) {
 		return Claims{}, "", fmt.Errorf("token expired")
 	}
 
-	if j.DisableXSRF {
+	if j.DisableXSRF || slices.Contains[[]string, string](j.XSRFIgnoreMethods, r.Method) {
```

Comment:
I don't think you need to implicitly specify types here as they should be inferred just fine. I believe `slices.Contains(j.XSRFIgnoreMethods, r.Method)` will work.

Reply:
Made this change. Thanks